sla_per_user_system_test.py

```diff
@@ -12,14 +12,18 @@
 # See LICENSE for more details.
 #
 # Copyright (c) 2016 ScyllaDB
-
+import logging
 import time
 
 from longevity_test import LongevityTest
 from sdcm.db_stats import PrometheusDBStats
+from sdcm.es import ES
 from test_lib.sla import ServiceLevel, Role, User
 
+logger = logging.getLogger(__name__)
```

Comment:
What's the purpose of using this logger? Most of log messages done by using `self.log` logger in this file

Also, please, use uppercase here (i.e., `LOGGER` instead of `logger`)

Reply:
You're right, this top level logger is not necessary - fixed by reverting to `self.log`.